TerrariaServerAPI/TerrariaApi.Server/ServerApi.cs

```diff
@@ -306,6 +310,12 @@ internal static void LoadPlugins()
 
 			List<FileInfo> fileInfos = new DirectoryInfo(ServerPluginsDirectoryPath).GetFiles("*.dll").ToList();
 			fileInfos.AddRange(new DirectoryInfo(ServerPluginsDirectoryPath).GetFiles("*.dll-plugin"));
+			if (AdditionalPluginsPath is string additionalPath)
```

Comment:
```suggestion
			if (AdditionalPluginsPath is not null)
```
imo, using a null check could express its purpose better. Reference: https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0150

Reply:
Applying this suggestion would break L315, would it not?

Reply:
> Applying this suggestion would break L315, would it not?

yes it would. the original code looks fine to me. if we actually turn on nullable contexts, the code here already tests for nulls

Reply:
so, is this fine as-is or nah?

Reply:
it's fine to me; external PRs require to approval buttons to merge though, so we're just waiting on one more approval

Reply:
I don't mind. LGTM